### PR TITLE
Changed approach for statuses page.

### LINKED ISSUE #264: The page should  be dependent on the selected status
If you go to https://google-books-stage.stanford.edu/statuses, then scroll to the bottom, select page 5, then change status to "Deposit Succeeded",  nothing will show up (because that status doesn't have 5 pages of results).  

Instead, the page should be dependent on the selected status.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #264

## Why was this change made?
To allow paging to work correctly for statuses.

## Was the documentation (README, DevOpsDocs, wiki) updated?
No.

### CODE REVIEW

File: app/models/status.rb [hunk 1]
```diff
@@ -7,11 +7,31 @@ def self.all
...
+  def self.find(id)
+    Settings.status_display.each do |vals|
+      status = new(vals.to_h)
+      return status if status.id == id
+    end
+    raise ActiveRecord::RecordNotFound
+  end
+
+  def self.find_by_status(status)
+    Settings.status_display.each do |vals|
```
Comment (jcoyne) [inline app/models/status.rb]: This could be `Enumerable#find`

Comment (justinlittman) [reply app/models/status.rb]: Fixed.

File: .rubocop.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ Metrics/AbcSize:
 
 Metrics/BlockLength:
   Exclude:
+    - app/models/status.rb
```
Comment (jcoyne) [inline .rubocop.yml]: Is this still necessary?

Comment (justinlittman) [reply .rubocop.yml]: No. Removed.

**Pull request merged**